Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -529,73 +529,60 @@ export async function getP12SHA1Hash(p12Path: string, p12Pwd: string): Promise<s
     openssl1.arg(['pkcs12', '-in', p12Path, '-nokeys', '-passin', 'pass:' + p12Pwd]);
 
     let openssl2: ToolRunner = tl.tool(opensslPath);
-    openssl2.arg(['x509', '-noout', '-fingerprint']);
+    openssl2.arg(['x509', '-noout', '-fingerprint', '-subject', '-dates']);
     openssl1.pipeExecOutputToTool(openssl2);
 
-    let sha1Hash: string;
-    openssl1.on('stdout', function (data) {
-        if (data) {
-            // find the fingerprint
-            data = data.toString().trim();
-            let fingerprint: string[] = data.match(/SHA1 Fingerprint=.+/g);
-            if (fingerprint && fingerprint[0]) {
-                sha1Hash = fingerprint[0].replace('SHA1 Fingerprint=', '').replace(/:/g, '').trim();
+    let fingerprint: string;
+    let commonName: string;
+    let notBefore: Date;
+    let notAfter: Date;
+
+    function onLine(line: string) {
+        if (line) {
+            const { key, value } = splitIntoKeyValue(line);
```

Comment:
Unfortunately this destructuring assignment isn't supported by the version of Node on our agents.  I suspect this is why the tests failed in the PR build.

Reply:
Thanks. I forgot to run `node make.js test --node 5 --task InstallAppleCertificateV2`

Reply:
Fixed.